src/server.js

```diff
@@ -236,15 +235,11 @@ class Server extends EventEmitter {
 
                     let parsed;
                     try {
-                        parsed = parseUrl(request.url);
+                        parsed = new URL(request.url);
                     } catch (e) {
                         // If URL is invalid, throw HTTP 400 error
                         throw new RequestError(`Target "${request.url}" could not be parsed`, 400);
                     }
-                    // If srcRequest.url is something like '/some-path', this is most likely a normal HTTP request
```

Comment:
This error was supposed to be helpful for debugging...

Reply:
Agree. I liked this error 😄

Reply:
This never executed because it throws `Target "/" could not be parsed` with `400`.

Reply:
Actually I saw it in the wild multiple times. Not too long ago even.

Reply:
```
$ node src/run_locally.js
Proxy server is running at http://127.0.0.1:8080
```

![image](https://user-images.githubusercontent.com/36894700/133284817-912777d6-21bf-4ace-92bd-594f9556de91.png)

Reply:
Still, I'm 100% sure I saw it in the wild. You don't imagine things like "I'm a HTTP proxy and not a regular web server" in error messages 😄 So there must've been some workflow where this actually triggered.

Reply:
> Still, I'm 100% sure I saw it in the wild.

Perhaps an outdated `proxy-chain` server? This is not possible even with master as `parseUrl` calls `new URL` and `new URL('/')` always throws, so the `if` is never reached.

Reply:
Yeah, you're right. It stopped working from 0.4.9 to 1.0.0.